examples/diff/index.html

```diff
@@ -83,20 +84,38 @@
       renderDiff();
     }
 
-    function keyboardInputHandler() {
+
+    /**
+     * Event handler for keyboard input.
+     * @param event The event object.
+     */
+    function keyboardInputHandler(event) {
       renderDiff();
       field.setValue(draftTextEl.value);
     }
 
+
+    /**
+     * Event handler triggered before the window unloads.
+     * @param event The event object.
+     */
+    function onBeforeUnloadHandler(event) {
+      window.removeEventListener('onbeforeunload', onBeforeUnloadHandler);
```

Comment:
Maybe not but I would consider it good practice to clean up listeners. I will also remove the keyboard listener!